tests/micro/hellojob.py

```diff
@@ -0,0 +1,19 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+@rfm.simple_test
+class VSCJobTest(rfm.RunOnlyRegressionTest):
+    descr = "test running job"
+    valid_systems = ["*"]
+    valid_prog_environs = ["*"]
```

Comment:
If you agree with my other proposed changes, this would be `valid_prog_environs = ["builtin"]`.

Reply:
seems legit